infra/cifuzz/fuzz_target.py

```diff
@@ -42,10 +42,9 @@
 BUFFER_TIME = 10
 
 # Log message if we can't check if crash reproduces on an recent build.
-COULD_NOT_TEST_ON_RECENT_MESSAGE = (
-    'Crash is reproducible. Could not run recent build of '
-    'target to determine if this code change (pr/commit) introduced crash. '
-    'Assuming this code change introduced crash.')
+COULD_NOT_TEST_ON_CLUSTERFUZZ_MESSAGE = (
+    'Could not run ClusterFuzz (old) build of target to determine if this '
```

Comment:
nit: this wording of "ClusterFuzz (old)" seems a bit confusing. Do we need to mention ClusterFuzz at all? 

Perhaps just reword as "Could not run on previous build of target" ?